Assets/Resources/Scripts/BreedingMenu.cs

```diff
@@ -0,0 +1,121 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BreedingMenu : MonoBehaviour
+{
+
+    public GameObject scrollMenuContent;
+    public GameObject droneImage;
+    public List<GameObject> allyDroneList;
+    public GameObject selectOneContent;
+    private bool selectOneSelecting;
+    public GameObject selectTwoContent;
+    private bool selectTwoSelecting;
+    public GameObject firstSelectedDrone;
+    public GameObject secondSelectedDrone;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        //Get list of living ally drones and populate scroll view with it
+        List<GameObject> droneList = GameObject.Find("Main Camera").GetComponent<Game>().livingAllies;
+        SetupDroneScrollView(droneList);
+
+        //Preset select buttons to not be selecting
+        DisableSelection();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        //While breeding menu is active, check if any drone in the list has been selected
+        foreach (var drone in allyDroneList) {
+            if (drone.GetComponent<DroneImage>().wasSelected) {
+                if (selectOneSelecting) { //Check if the first select button was active
+                    if (firstSelectedDrone != null) {
+                        RemoveDroneFromSelectOne();
+                    }
+                    AddDroneToSelectOne(drone);
+                    DisableSelection();
+                } else if (selectTwoSelecting) { //Check if the second select button was active
+                    if (secondSelectedDrone != null) {
+                        RemoveDroneFromSelectTwo();
+                    }
+                    AddDroneToSelectTwo(drone);
+                    DisableSelection();
+                }
+                drone.GetComponent<DroneImage>().WasUnselected();
+            }
+        }
+    }
+
+    public void Breed() {
+
+        DisableSelection();
+
+        if (firstSelectedDrone == null || secondSelectedDrone == null) {
+            //TODO: Graceful handling for this situation
+            Debug.Log("Didn't have first or second drone selected when trying to breed");
+            return;
+        }
+
+        GameObject.Find("Main Camera").GetComponent<Game>().breedingTarget1 = firstSelectedDrone.GetComponent<DroneImage>().linkedDrone.GetComponent<Drone>();
+        GameObject.Find("Main Camera").GetComponent<Game>().breedingTarget2 = secondSelectedDrone.GetComponent<DroneImage>().linkedDrone.GetComponent<Drone>();
+        GameObject.Find("Main Camera").GetComponent<Game>().Breed();
```

Comment:
it may be cleaner if I make the breeding targets an argument of the function rather than fields